core/src/main/java/com/novoda/noplayer/internal/exoplayer/SkippedAdverts.java

```diff
@@ -32,6 +32,21 @@ static AdPlaybackState markAllNonPlayedAdvertsAsSkipped(List<AdvertBreak> advert
         return adPlaybackStateWithSkippedAdGroups;
     }
 
+    /**
+     * Transforms all adverts in the given break that are not currently Played to Skipped.
+     *
+     * @param advertBreaks    The client representation of all the adverts break.
+     * @param advertBreak    The client representation of the advert break to be skipped.
```

Comment:
I think we need to run the auto-formatter on this class, this alignment looks a bit out.